BAH-2181 | Added env for openmrs container

atish160384 opened the pull request:
- Added few environment variables for openmrs container to be fetched in appointments module

gsluthra commented on bahmni-lite/.env:
```diff
@@ -9,6 +9,8 @@ OPENMRS_HOST=openmrs
 OPENMRS_PORT=8080
 OPENMRS_ATOMFEED_USER=admin
 OPENMRS_ATOMFEED_PASSWORD=Admin123
+OPENMRS_SYSTEM_SMS_USER=systemuser
```
Can we plz make this user as `system-user`.

atish160384 replied:
Sure will do the changes

mohan-13 commented:
Hello, I would like to get a bit of context here.  Why does the OpenMRS container in itself needs the HOST and PORT parameter ? Why do we need credentials from within the OMRS application context itself?

atish160384 commented:
> Hello, I would like to get a bit of context here. Why does the OpenMRS container in itself needs the HOST and PORT parameter ? Why do we need credentials from within the OMRS application context itself?

So Mohan I had added them for scheduler which needs to send message to the patients having appointment for sending the message we need a reporting_session cookie to (in sms-service) and to extract that reporting session (of system-user) we would need to Authenticate and login to openmrs first for which I added these Host and Port
